mime/multipart.ts

```diff
@@ -276,13 +283,15 @@ export class MultipartReader {
   }
 
   /** Read all form data from stream.
-   * If total size of stored data in memory exceed maxMemory,
+   * If total size of stored data in memory exceed options.maxMemory,
    * overflowed file data will be written to temporal files.
    * String field values are never written to files.
    * null value means parsing or writing to file was failed in some reason.
-   * @param maxMemory maximum memory size to store file in memory. bytes. @default 10485760 (10MB)
+   * @param options options to configure the behavior of storing
+   * overflow file data in temporal files.
    *  */
-  async readForm(maxMemory = 10 << 20): Promise<MultipartFormData> {
+  async readForm(options?: ReadFormOptions): Promise<MultipartFormData> {
```

Comment:
To protect the current usage of this method, we'd like to support the current signature as well. I think we can use overload feature like the below to support both the current and suggested interface:

```ts
  async readForm(maxMemory: number): Promise<MultipartFormData>;
  async readForm(options?: ReadFormOptions): Promise<MultipartFormData>;
  async readForm(maxMemoryOrOptions: number | ReadFormOptions): Promise<MultipartFormData> {
    ...
  }
```

Reply:
How should the documentation reflect this? Should `@param` last function definition?